set.mm

```diff
@@ -494109,6 +494139,24 @@ such that every prime ideal contains a prime element (this is a
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  ${
+    $d M p $.  $d p ph $.
+    0ringmon1p.1 $e |- M = ( Monic1p ` R ) $.
+    0ringmon1p.2 $e |- B = ( Base ` R ) $.
+    0ringmon1p.3 $e |- ( ph -> R e. Ring ) $.
+    0ringmon1p.4 $e |- ( ph -> ( # ` B ) = 1 ) $.
+    $( There are no monic polynomials in a zero ring.  (Contributed by Thierry
+       Arnoux, 5-Feb-2025.) $)
+    0ringmon1p $p |- ( ph -> M = (/) ) $=
```

Comment:
"polynomials in a zero ring" -> "polynomials over a zero ring"

Reply:
Done